tb/cocotb/gpu/test_random_kernels.py

```
@@ -50,25 +58,31 @@
-    k.emit(vmov_tid_x(R_TID))          # r1[l] = l
-    k.emit(vaddi(R_TWO, 0, 2))         # r2 = 2
-    k.emit(vsll(R_OFF, R_TID, R_TWO))  # r3[l] = l*4
 
-    # seed working registers with random constants
-    for r in WORKING:
-        k.emit(vaddi(r, 0, rng.randint(-2048, 2047)))
+def _seed_input_region(rng: random.Random) -> dict:
+    """Return a dict {addr: value} seeding INPUT_REGION for one kernel."""
+    return {
+        INPUT_REGION_BASE + l * 4: rng.randint(0, 0xFFFF_FFFF)
+        for l in range(INPUT_REGION_SIZE)
```

Flake8 (7.3.0)

[error] ambiguous variable name 'l'

(E741)